.env.example

```diff
@@ -2,3 +2,6 @@ MNEMONIC=test test test test test test test test test test test junk
 INFURA_KEY=
 ALCHEMY_KEY=
 ETHERSCAN_KEY=
+
+// Options: mainnet, avalanche, polygon, bsc, fantom
```

Comment:
would be good to be more descriptive here to explain what this key is used for (i.e. setting the forking network when running unit tests for ante test on a forked chain), also probably good to have a default value set as it will error if left blank